elo_boat.py

```diff
@@ -55,6 +51,18 @@
 trueenv.make_as_global()
 
 
+def is_mention(text):
+    if "<@" in text and "<@&" not in text and any(i.isdigit() for i in text):
+        return True
+    return False

```

Comment:
looks trick-able, consider calling discord API to confirme it's a real mention

Reply:
changed it to is_metion_format